docroot/themes/humsci/humsci_basic/src/scss/ckeditor/imports.scss

```
@@ -322,6 +322,21 @@ $px-only: true;
       }
     }
   }
+
+  // Fix Video preview in CKeditor for "Align center and break text" option
+  .drupal-media-style-align-center {
+    max-width: none;
+
+    [class*="hb-media"]:not(.hb-media-video) {
```

I'm pretty sure all media elements use the `<article>` tag. In that case, this rule can be simplified:

```suggestion
    article:not(.hb-media-video) {
```